config/samples/bpfman.io_v1alpha1_bpfapplication.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: bpfman.io/v1alpha1
```

Comment:
I want to make sure we test every program type before merging this pr.  I was thinking that this would be a good place to do it.

Reply:
we can add more post merge